client/src/ui-components/ShoppingCartComponent/ShoppingCartOrderSummary.js

```diff
@@ -0,0 +1,137 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./ShoppingCartOrderSummaryCSS.css";
+import React from "react";
+import { Container, Col, Row, Card, Form, FloatingLabel} from "react-bootstrap";
+import  Divider from "../Divider/Divider";
+import Button from "../Button/Button";
+import { OrderRecapRow } from "./OrderRecapRow";
+
+function checkboxShipmentComponent(props){
+    return <div className="form-check">
+                <input onChange={()=>{
+                    if(props.deliveryType==="Pickup" || props.deliveryType===""){//i.e. if from "pickup" to "shipment", then clear the address field
+                        props.setDeliveryAddress("")
+                        props.setAmount(props.amount + 20);
```

Comment:
it's better to pass a fee variable as props, instead of just adding 20. So in the future it will be easier to update